spec/controllers/reports_controller_spec.rb

```diff
@@ -2,20 +2,21 @@
 
 describe ReportsController, type: :controller do
   before(:all) do
```

Comment:
Let's leave it for #1412, it seems like we've resolved the original issue (hopefully) so we can work through the whole test suite systematically over the next few months.